app/controllers/v1/charts_controller.rb

```diff
@@ -37,6 +37,14 @@ def destroy
     head :no_content
   end
 
+  def clone
+    authorize! :clone, Chart
+
+    chart = Chart.find_by(id: params[:id]).clone
+
+    render json: serialized_response(chart), status: :created
+  end
```

Comment:
I think we can remove this code. For some reason, we included `Resource::Clone` in this controller.